parser-typechecker/src/Unison/Codebase/TranscriptParser.hs

```diff
@@ -111,6 +112,7 @@ run dir configFile stanzas codebase = do
     numberedArgsRef          <- newIORef []
     inputQueue               <- Q.newIO
     cmdQueue                 <- Q.newIO
+    unisonFiles              <- newIORef Map.empty
```

Comment:
Could you explain `unisonFiles` and how this works?  Which commands depend on it?

Reply:
Explain in code comments? 

`unisonFiles` stores the contents of each of the unison stanzas, keyed by "filename", so that the `load` command works as expected in transcripts.

Reply:
Oh wow. Good thinking.  I didn't notice that it was specific to the transcript runner.